subgraphs/playback/src/utils/schema.ts

```diff
@@ -0,0 +1,51 @@
+import { FieldNode, GraphQLResolveInfo, Kind, SelectionNode, SelectionSetNode } from 'graphql';
+
+export const shouldLoadFullArtist = (
+  pathToArtist: string[],
+  info: GraphQLResolveInfo
+) => {
+  return ['followers', 'genres', 'images', 'popularity'].some((fieldName) =>
+    selectsField([...pathToArtist, fieldName], info)
+  );
+};
+
+export const selectsField = (path: string[], info: GraphQLResolveInfo) => {
```

Comment:
Looks like these functions are duplicates of [`src/utils/graphql.ts`](https://github.com/apollographql/spotify-showcase/pull/66/files?diff=unified&w=1#diff-d90aba0bad98da6a35018e48b47bd5b1b3aa619bb9192e3263d116bb9d1e4ab6). Any chance we could just use those and remove from this file?

Reply:
Yes...I don't even know why that file is there as it isn't referenced anywhere 🙃